flybywiresim/a32nx
# fix: Fix issues with update throttler

Fixes #3398

## Summary of Changes
- Ensure that _deltaTime that gets passed into the throttler is accumulated in case of
lower instrument refresh rate setting.
- Update map more frequently when turning.

## Testing instructions
- Load any flight plan.
- Do some turns and fly along
- Observe how flight plan update is smoother.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
